ding/model/wrapper/model_wrappers.py

```diff
@@ -776,6 +777,8 @@ def __init__(
         self.action_range = action_range
 
     def forward(self, *args, **kwargs):
+        sigma = kwargs.pop('sigma')
+        self.noise_generator = GaussianNoise(mu=0, sigma=sigma)
```

Comment:
modify here as follows:
```python
sigma = kwargs.pop('sigma')
if sigma is not None:
    self.noise_generator.sigma = sigma
```
And add a `property` and `setter` method for sigma of `GaussianNoise`